src/consts.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Constants used by the nginx-ingress-integrator charm."""
+
+CREATED_BY_LABEL = "nginx-ingress-integrator.charm.juju.is/managed-by"
```

Comment:
Can't these be defined in the other files for improved cohesion?

Reply:
Yep, I removed the consts file altogether